apps/prairielearn/assets/scripts/question.ts

```diff
@@ -323,11 +289,17 @@ function setupDynamicObjects() {
   }
 }
 
-function loadPendingSubmissionPanel(this: HTMLDivElement) {
+async function loadPendingSubmissionPanel(this: HTMLDivElement, includeScorePanels: boolean) {
```

Comment:
Since all calls to this function have a custom `this`, wouldn't it make sense to make it a regular parameter instead?

Reply:
Absolutely, see https://github.com/PrairieLearn/PrairieLearn/pull/10792#discussion_r1794413856:

> I want to switch to using an actual property on the event instead of relying on jQuery's binding of this. Maybe I can just rip out jQuery here entirely?

You're correct that I can't rip out jQuery, but I bet I can use `e.target` or something here. I'll experiment.

Reply:
That worked; done in 99b7353.